pingcap/tidb
# server: reset current start ts in processlist when transaction committed

Signed-off-by: Shuaipeng Yu <jackysp@gmail.com>



### What problem does this PR solve? 
If
```sql
begin;
commit;
```
the current start ts won't be reset when `select * from information_schema.processlist;`

### What is changed and how it works?
Reset it if the transaction is committed.

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)
1. start session1,
```sql
begin;
```
2. start session2, 
```sql
select * from information_schema.processlist;
```
3. session1,
```sql
select * from t;
```
4. session2,
```sql
select * from information_schema.processlist;
```
the current start ts should not be null
5. session1,
```sql
commit;
```
6. session2,
```sql
select * from information_schema.processlist;
```
the current start ts should be null.

Code changes

 - Has exported function/method change

Side effects

 - Increased code complexity

Release note

 - Write release note for bug-fix or new feature.
